sql/npc_list.sql

```diff
@@ -21582,14 +21582,14 @@ INSERT INTO `npc_list` VALUES (17523346,'blank','     ',0,0.000,0.000,0.000,0,50
 INSERT INTO `npc_list` VALUES (17523347,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17523348,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17523349,'qm','???',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
--- NC: INSERT INTO `npc_list` VALUES (17523350,'NOT_CAPTURED','Volto Oscuro',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17523350,'Volto_Oscuro','Volto Oscuro',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x010002038411E620E630E640E650406100700000,32,NULL,0);
 INSERT INTO `npc_list` VALUES (17523351,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x010002038411E620E630E640E650406100700000,32,NULL,1);
```

Comment:
Did you account for shifts when looking for that reference?  It seems odd that we'd be going from a modeled NPC here to a blank one.